docs/_posts/0000-05-01-oo-inheritance.md

```diff
@@ -222,7 +225,72 @@ _"A steering wheel **is not** a kind of car"_ - **X**
 </p>
 </div>
 
+--
+
+> Favor object composition over class inheritance.
+
+—[Design Patterns: Elements of Reusable Object-Oriented Software](https://uk.bookshop.org/books/design-patterns-elements-of-reusable-object-oriented-software/9780201633610)  
+by Erich Gamma, Richard Helm, Ralph Johnson, John Vlissides  
+(AKA the ‘Gang of Four’ book)
+
+--
+
+> **Tight coupling**. Classes that are tightly coupled are hard to reuse in isolation, since they depend on each other. Tight coupling leads to monolithic systems, where you can’t change or remove a class without understanding and changing many other classes. The system becomes a dense mass that’s hard to learn, port, and maintain.
 
+—[Design Patterns: Elements of Reusable Object-Oriented Software](https://uk.bookshop.org/books/design-patterns-elements-of-reusable-object-oriented-software/9780201633610)  
+
+--
+
+### Class Inheritance vs Object Composition
+
++ **Class Inheritance**: Classes inherit behaviour from parent class <span style="color:red">(tightest coupling)</span>
+  + **White-box reuse**—often the internals of parent classes are visible to subclasses
```

Comment:
I think the white-box vs black-box stuff is just a bit too detailed and will only serve to confuse.